### ISSUE TITLE
Replace apply() with function call in twisted/trial/_dist/distreporter.py

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8557]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8557|
|Type|enhancement|
|Created|2016-07-04 08:48:46Z|
|Branch|https://github.com/twisted/twisted/tree/8557-rodrigc-trial|

The apply() builtin in Python 2 has been removed in Python 3.
It should be replaced by a function call which works on Python 2 and 3.
See:
http://python3porting.com/differences.html#apply

<details><summary>Searchable metadata</summary>

```
trac-id__8557 8557
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__8557_rodrigc_trial 8557-rodrigc-trial
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1467622126929610 1467622126929610
changetime__1467625570215875 1467625570215875
version__None None
owner__rodrigc rodrigc

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_1"></a>|@rodrigc commented|
|-|-|

See:
https://github.com/twisted/twisted/pull/289

**Comment by twisted-trac** [comment]
|[<img alt="Lukasa's avatar" src="https://avatars.githubusercontent.com/u/1382556?s=50" width="50" height="50">](https://github.com/Lukasa)<a name="note_2"></a>|@Lukasa set owner to @rodrigc|
|-|-|

Well this was the easiest review ever. LGTM, go forth and merge.

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|GitHub set status to `closed`|
|-|-|

In changeset 4e139fe239ff0fcc288a87460d772feca2b1ae8c
```
#!CommitTicketReference repository="" revision="4e139fe239ff0fcc288a87460d772feca2b1ae8c"
Merge pull request #289 from twisted/8557-rodrigc-trial

Author: rodrigc
Reviewer: Lukasa
Fixes: #8557

Replace apply() with a function call. apply() is gone in Python 3
```
